src/main/java/fr/techad/edc/client/internal/io/HttpReaderImpl.java

```diff
@@ -279,35 +279,43 @@ private DocumentationItemType getType(JsonObject jsonObject) {
 
     }
 
-    private Map<String, String> readLabelsForLang(String languageCode) throws IOException, InvalidUrlException {
-        Map<String, String> labels = Maps.newHashMap();
-        String label;
-
+    private String getLabelUrl(String languageCode) throws InvalidUrlException {
         String relativePath = POPOVER_I18N_PATH + languageCode + I18N_FILE_EXTENSION;
         String labelUrl = StringUtils.appendIfMissing(this.clientConfiguration.getDocumentationUrl(), "/") + relativePath;
         LOGGER.debug("Reading labels for lang {}, url {}, labelUrl {}", languageCode, relativePath, labelUrl);
+        return labelUrl;
+    }
+
+    private I18NContent readi18NContent(String languageCode) throws IOException, InvalidUrlException {
```

Comment:
Change this declaration like this:
**private void readI18NContent(String languageCode, I18NContent i18nContent) {...}**